index.html

```diff
@@ -32,13 +38,194 @@ <h2>I'm a Software Developer</h2>
+        <div class="project">
+          <div class="head_card">
+            <h2>Professional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+        <div class="project">
+          <div class="head_card">
+            <h2>Professional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+        <div class="project">
+          <div class="head_card">
+            <h2>Professional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+        <div class="project">
+          <div class="head_card">
+            <h2>Professional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+        <div class="project">
+          <div class="head_card">
+            <h2>Professional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about_me">
+      <div class="about">
+        <h1>About me</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button>Get My Resume</button>
```

Comment:
- [ ] kindly add the button styling: Active, Hover, and Pressed as mentioned in the requirements and Figma design, kindly make sure to apply the styling to all of your buttons 🙏 

![image](https://user-images.githubusercontent.com/89282221/198715234-389e61ee-519c-4cf0-844f-fc189a17d7d4.png)